### ISSUE TITLE
Please add a screenshot with command docstrings shown to the README

### ISSUE THREAD

**Issue by piotrklibert** [justbur/emacs-which-key#332]
I'm not sure if there are many people like me, but I avoided using this project for years because it seemed pretty useless to me, based on the screenshots and description in the README. At the end of last year, I did a major overhaul of my Emacs config and decided to take a closer look at the customization options. There I discovered that which-key is able to also display docstrings - and **that** is immensely more useful to me than displaying just the command names. Coupled with the ability to set columns' widths and paging, it gives a wonderful, quick overview of what will really happen when I press some key. I think "seeing is believing" has a certain amount of truth to it, so if there are others like me, including a screenshot like this will help them decide to use which-key:

![screenshot_2022-01-08_1103](https://user-images.githubusercontent.com/1244294/148640196-4f565315-f0b4-482e-830d-b9b3e383ab14.png)

**Comment by justbur** [comment]
I can do that. Do I have your permission to use this one?

**Comment by piotrklibert** [comment]
@justbur Obviously, yes! Sorry, I didn't get the notification for some reason, I'm cleaning up
